src/ProjectConfiguration.ts

```diff
@@ -213,6 +213,13 @@ function constrainedSetDiagnostics({
   return diagnostics;
 }
 
+function normalizeLanguageTag(tagString: string) {
```

Comment:
`tagString` is redundant as we have type `string` associated to it. maybe `languageTag`?